assets/stylesheets/_main.scss

```diff
@@ -70,6 +70,10 @@ body {
 	}
 }
 
+.layout {
+	background: var( --color-surface-backdrop );
```

Comment:
Added this variable to control the new background color behind surfaces that's introduced with Laser Black.

Reply:
this is the bit that broke gutenberg-in-calypso https://github.com/Automattic/wp-calypso/pull/29147#issuecomment-446569432

we do want to keep this, but we'll need to add a bgcolor override somewhere down the chain.